### PR TITLE
Minor CI improvements, Rename submit_event to record_event

### PR DESCRIPTION
**Pull request by Guillemdb**
This is  a PR addressing some minor issues with the CI:

- Requiring at least python 3.5.: https://github.com/src-d/confident-metrics/pull/1#discussion_r283085016
- Renaming submit_event: Im addressing the Issue here because the metric recording is implemented in lookout-sdk-ml
https://github.com/src-d/ml-core/issues/19

- Naming CI stages https://github.com/src-d/ml-backlog/issues/63

- Updating the modelforge version to 0.13.2

### CODE REVIEW

File: lookout/core/metrics.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def _child_samples(self):
         )
 
 
-def submit_event(key: str, value: Union[int, float, bool], description: str = ""):
+def record_event(key: str, value: Union[int, float, bool], description: str = ""):
```
Comment (vmarkovtsev) [inline lookout/core/metrics.py]: Looks like we don't have any tests for this function, otherwise we would have to edit the tests too. This is sad 😢

Comment (Guillemdb) [reply lookout/core/metrics.py]: Im fixing this. I forgot to add the files where the function is referenced.

Comment (Guillemdb) [reply lookout/core/metrics.py]: Solved

File: Untitled.ipynb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (vmarkovtsev) [inline Untitled.ipynb]: This should be removed

Comment (Guillemdb) [reply Untitled.ipynb]: Ups, I thought notebooks were included on the gitignore. Already solved

**Comment by Guillemdb** [comment]
Given that someone already bumbped the modelforge version, Im gonna remove the commit.

**Comment by vmarkovtsev** [comment]
Perfect! The commits in this PR are sexy now 👍

**Pull request merged**